config/dev.exs

```diff
@@ -2,5 +2,5 @@ import Config
 
 config :still,
   dev_layout: true,
-  base_url: "http://localhost:3000",
+  base_url: "",
```

Comment:
Is this intentional? If so, is it _required_?

Reply:
Yeah :/ the base_url should be the path, not the full URL. We need to know that for the router to know which file to recompile.

Reply:
This was removed from the other envs, though, should it be included?

Reply:
Yeah, it can be removed :+1: